client/reducers/foods.js

```diff
@@ -1,10 +1,19 @@
 import { GET_FOODS_SUCCESS } from './../actions/foods'
+import { GET_CATEGORIES_SUCCESS } from './../actions/categories'
```

Comment:
You should be using the action type from `/actions/category`, not `/actions/categories`.